src/regexbench.cpp

```diff
@@ -44,6 +46,9 @@ int main(int argc, const char *argv[]) {
     case ENGINE_REMATCH:
       engine = std::make_unique<regexbench::REmatchEngine>();
       break;
+    case ENGINE_PCRE2:
```

Comment:
Sort `case`'s alphabetically unless there is more logical grouping of cases or the alphabetical ordering affects performance.